CHANGELOG_UNRELEASED.md

```diff
@@ -30,6 +30,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
   `big_enum_val_cond`, `big_enum_rank_cond`,
   `big_enum_val`, `big_enum_rank`, `big_set`.
 
+- Natural arithmetic theorems: `modn_divl` and `divn_modl`.
```

Comment:
I do not know what "natural" means. I just write "new arithmetic theorems".

Reply:
fixed